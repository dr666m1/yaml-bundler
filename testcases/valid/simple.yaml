foo: !include ./sub/simple.yaml
bar: !include
  filepath: ./sub/simple.yaml
