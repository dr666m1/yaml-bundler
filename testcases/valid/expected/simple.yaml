foo:
  foo: 1
  bar: 2
bar:
  foo: 1
  bar: 2
